src/lib/journey/stages/one-time-password.svelte

```diff
@@ -53,6 +60,24 @@
 
   $: {
     formMessageKey = convertStringToKey(form?.message);
+
+    const confirmationCallbacks = step.getCallbacksOfType(CallbackType.ConfirmationCallback);
+    if (confirmationCallbacks.length) {
+      const confirmationCb = confirmationCallbacks[0] as ConfirmationCallback;
+      buttons = confirmationCb
+        .getOptions()
+        .map((option, index) => ({ value: `${index}`, text: option }));
+    }
+
+    /**
+     * Filter out ConfirmationCallbacks; we'll use them seperately
+     */
+    modifiedCallbacks = step.callbacks.filter((callback) => {
+      if (callback.getType() === CallbackType.ConfirmationCallback) {
+        return false;
+      }
+      return true;
+    });
```

Comment:
This helped me clean up the `ConfirmationCallback` component, and I'm thinking of doing a similar thing with `PollingWaitCallback` component.